src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -83,7 +83,7 @@ public sealed class SonarQubeIssuesProvider : ISonarQubeIssuesProvider
             refreshTimer.Elapsed += OnRefreshTimerElapsed;
 
             initialFetchCancellationTokenSource = new CancellationTokenSource();
-            this.initialFetch = Task.Factory.StartNew(DoInitialFetchAsync, initialFetchCancellationTokenSource.Token);
+            this.initialFetch = Task.Factory.StartNew(DoInitialFetchAsync, initialFetchCancellationTokenSource.Token).Unwrap();
```

Comment:
Ah. I suspect that the "Unwrap" here explains the problem you were seeing before. `DoInitialFetchAsync` is returning a Task, so `Task.Factory.StartNew(DoInitialFetchAsync)` is returning a Task that returns a Task. Can we simplify this?